conduct.md

```diff
@@ -134,14 +134,14 @@ TODO: Write an introductory paragraph.
 
 TODO: Write an introductory paragraph.
 
-#### Acceptable Conduct
+#### Level 1 Infraction
 
-- TODO
-- TODO
-- TODO
+Students who commit violation once will be recorded by professor and warned orally.
 
-#### Unacceptable Conduct
+#### Level 2 Infraction
 
-- TODO
-- TODO
-- TODO
+Students who commit violation 3 times or more in total will be required to have a meeting with professor to talk about their current infractions and develope a plan to avoid future ones.
+
+#### Level 3 Infraction
+
+Students who commit violation 5 times in total or avoid meeting with professor after a level 2 infraction will be deducted grades on their current assignment.
```

Comment:
Grammatically, this should be "will have points deducted from their grade on the current assignment."